ethcore/sync/src/chain/mod.rs

```diff
@@ -717,6 +734,7 @@ impl ChainSync {
 			snapshot: Snapshot::new(),
 			sync_start_time: None,
 			transactions_stats: TransactionsStats::default(),
+			unfetched_pooled_transactions: Default::default(),
```

Comment:
```suggestion
			unfetched_pooled_transactions: H256FastMap::default(),
```
(to me it's more readable to see the actual type, but maybe it's just me)